### PR TITLE
Add gho ccip addresses

### PR DESCRIPTION
**Pull request by CheyenneAtapour**
Added the following contract addresses:

- ETH_TOKEN_POOL = 0x5756880B6a1EAba0175227bf02a7E87c1e02B28C; 

- ARB_GHO = 0x7dfF72693f6A4149b17e7C6314655f6A9F7c8B33;

- ARB_TOKEN_POOL = 0xF168B83598516A532a85995b52504a2Fa058C068;

### CODE REVIEW

**Comment by sakulstra** [comment]
Hello @CheyenneAtapour, adding `ARB_GHO` doesn't really make sense - it will be added automatically once listed on the pool.

For the other two addresses, they should probably be on misc, not on the pool as it has zero relation to the pool as i understand.

**Comment by eboadom** [comment]
Also better to rename `ETH_TOKEN_POOL` and `ARB_TOKEN_POOL` for something more explanatory, at least indicating that it is related to GHO (maybe with CCIP too)

File: scripts/configs/types.ts [hunk 1]
```diff
@@ -102,6 +102,8 @@ export interface NetworkAddresses<T extends Record<string, AddressInfo> = {}> {
     AAVE_CL_ROBOT_OPERATOR?: Hex;
     GHO_TOKEN?: Hex;
     GHO_FLASHMINTER_FACILITATOR?: Hex;
+    GHO_ETH_CCIP_POOL?: Hex;
+    GHO_ARB_CCIP_POOL?: Hex;
```
Comment (miguelmtzinf) [inline scripts/configs/types.ts:106]:
```suggestion
    GHO_CCIP_TOKEN_POOL?: Hex;
```

File: scripts/configs/networks/arbitrum.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export const arbitrumAddresses: NetworkAddresses<{
   name: 'Arbitrum',
   chainId: ChainId.arbitrum_one,
   addresses: {
+    GHO_ARB_CCIP_POOL: '0xF168B83598516A532a85995b52504a2Fa058C068',
```
Comment (miguelmtzinf) [inline scripts/configs/networks/arbitrum.ts:12]: move down to the end of the object

Comment (miguelmtzinf) [reply scripts/configs/networks/arbitrum.ts:12]: add `GHO_TOKEN` right before

Comment (CheyenneAtapour) [reply scripts/configs/networks/arbitrum.ts]: done

File: scripts/configs/types.ts (hunk 1 above)
Comment (CheyenneAtapour) [reply scripts/configs/types.ts]: done

**Comment by miguelmtzinf** [comment]
Thanks @CheyenneAtapour . Hope is looking good now @eboadom @sakulstra ! Thanks

**Comment by sakulstra** [comment]
No, like I commented bwfore GHO token should not be here - it will be added via assets automatically once it's listed.

**Review by sakulstra — APPROVED** [review]

**Pull request merged**